AndroidNativeLoginTemplate/template.js

```diff
@@ -48,6 +48,7 @@ function prepare(config, replaceInFiles, moveFile, removeFile) {
     var templateMainActivityFile = path.join('app', 'src', 'main', 'java', 'com', 'salesforce', 'androidnativelogintemplate', 'MainActivity.kt');
     var templateMainApplicationFile = path.join('app', 'src', 'main', 'java', 'com', 'salesforce', 'androidnativelogintemplate', 'MainApplication.kt');
     var templateNativeLoginFile = path.join('app', 'src', 'main', 'java', 'com', 'salesforce', 'androidnativelogintemplate', 'NativeLogin.kt');
+    var templateNativeLoginViewModelFile = path.join('app', 'src', 'main', 'java', 'com', 'salesforce', 'androidnativelogintemplate', 'NativeLoginViewModel.kt');
```

Comment:
For templates with many source files we can find them programmatically (like we did [here](https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/dev/MobileSyncExplorerKotlinTemplate/template.js#L38)).
But I guess 4 source files is not too bad.